babyagi.py

```diff
@@ -67,11 +69,11 @@ def get_ada_embedding(text):
     text = text.replace("\n", " ")
     return openai.Embedding.create(input=[text], model="text-embedding-ada-002")["data"][0]["embedding"]
 
-def openai_call(prompt: str, use_gpt4: bool = False, temperature: float = 0.5, max_tokens: int = 100):
-    if not use_gpt4:
-        #Call GPT-3 DaVinci model
+def openai_call(prompt: str, temperature: float = 0.5, max_tokens: int = 100):
+    if not USE_CHAT_COMPLETIONS:
```

Comment:
I'd prefer if we keep the parameter here. Even though it's not really properly used right now, having the parameter allow us to build a hybrid mode where some of the calls are made with GPT-3 models, and others with GPT-4 chat completions.

Reply:
We can just do that when we need it. Right now it wasn’t being used and the point of this PR is to clean it up and allow changing the model at least globally.

Reply:
In other words, when we want to build the use case you're describing, it will be easier to thread it in directly in the way that applies then.

Reply:
Added the parameter to this method only